sdk/core/Azure.Core/src/HttpMessage.cs

```diff
@@ -33,6 +33,8 @@ public HttpMessage(Request request, ResponseClassifier responseClassifier)
             Request = request;
             ResponseClassifier = responseClassifier;
             BufferResponse = true;
+            OperationStartTime = DateTimeOffset.UtcNow;
+            ProcessingContext = new ProcessingContext(this);
```

Comment:
We should not have a field that stores the context. It just makes the class larger. Instead, the property that returns the context, should just return "new ProcessingContext(this)"